model/model.py

```diff
@@ -0,0 +1,49 @@
+"""Define the main MeshGraphNet model"""
+
+import numpy as np
+import torch
+
+from meshnets.datatypes.graph import Graph
+from meshnets.model.encoder import GraphEncoder
+from meshnets.model.decoder import GraphDecoder
+from meshnets.model.processor import GraphProcessor
+
+
+class MeshGraphNet(torch.nn.Module):
+    """MeshGraphNet model.
+    
+    It is composed of a GraphEncoder, a GraphProcessor and a GraphDecoder.
```

Comment:
Suggestion: "The model architecture is composed of graph Encoder, Processor and Decoder.
The input is a Graph object which is encoded together with its features to a latent representation. Thereafter, message passing steps are applied by the Processor on the latent Graph object and, at the end, the Decoder outputs features at each node."